{"uuid": "6c451d24-4a45-4a83-a3b6-7a2db3d46873", "children": ["3e9102d3-359f-4def-9c0b-a26924522cde", "cd7fa81c-daa5-4058-a286-a8021bad1d10", "3017f3bc-8512-4b6f-a0c1-d8c1accb7146", "997a1b5d-a3ff-4396-a8e5-b1fa4c048721", "1ca78b11-0715-4207-85e8-fc25a3e46f84"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 30, in setup\n    peviitor_jobs_data = TestScrapers.setup_tests.get_jobs_careers(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 17, in get_jobs_careers\n    self.scraper_data = self.jobdetails.scrape_peviitor(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 91, in scrape_peviitor\n    response_data = JobDetails._get_request(company_name)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 47, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/peviitor_update.py\", line 37, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.laurentiumarian.ro/get_token\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1725734293293, "stop": 1725734294134}], "start": 1725734293293, "stop": 1725734294342}